packages/opentelemetry-core/src/trace/sampler/TraceIdRatioBasedSampler.ts

```diff
@@ -44,4 +47,14 @@ export class TraceIdRatioBasedSampler implements Sampler {
     if (typeof ratio !== 'number' || isNaN(ratio)) return 0;
     return ratio >= 1 ? 1 : ratio <= 0 ? 0 : ratio;
   }
+
+  private _accumulate(traceId: string): number {
```

Comment:
A private function which doesn't use `this` might be better off as a regular `function _accumulate(traceId: string): number` in order to better support minification

Reply:
I think it would be better to open an different issue to improve minification in the core repo, web instrumentations also use `private` a lot

Reply:
I've opened one https://github.com/open-telemetry/opentelemetry-js/issues/2041